src/platform/packages/shared/kbn-unified-metrics-grid/src/components/metric_flyout/overview_tab.tsx

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { EuiBadge, EuiText, EuiSpacer, EuiDescriptionList } from '@elastic/eui';
+import React from 'react';
+import { DimensionBadges } from './dimension_badges';
+import type { MetricField } from '../../types';
+import { categorizeDimensions } from '../../utils';
+import { TabTitleAndDescription } from './tab_title_and_description';
+
+interface OverviewTabProps {
+  metric: MetricField;
+}
+
+export const OverviewTab = ({ metric }: OverviewTabProps) => {
+  const { requiredDimensions, optionalDimensions } = categorizeDimensions(
+    metric.dimensions || [],
+    metric.name
+  );
+  return (
+    <>
+      <TabTitleAndDescription metric={metric} />
+      {/* OpenTelemetry semantic conventions */}
+      {metric.source === 'otel' && metric.stability && (
```

Comment:
Could we type `source` with the source types it accepts?

Reply:
Do we know all possible sources? I see 'otel' | 'ecs' | 'custom'  in the [PoC](https://github.com/elastic/metrics-experience-prototype/blob/main/src/server/routes/metrics_fields/get_metrics_fields.ts#L475-L477)  but in the current code version we don't have otel: https://github.com/jennypavlova/kibana/blob/231607-metricsdiscover-add-the-metrics-flyout-to-the-grid-charts/src/platform/plugins/shared/metrics_experience/server/routes/fields/get_metric_fields.ts#L102 - this could be a bug or on purpose, idk 🤔